net.kvirc.KVIrc.yaml

```
@@ -0,0 +1,24 @@
+app-id: net.kvirc.KVIrc
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-appdata-file: net.kvirc.KVIrc5.appdata.xml
+rename-desktop-file: net.kvirc.KVIrc5.desktop
+rename-icon: kvirc
+command: kvirc
+
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --filesystem=xdg-download
+  - --persist=.config/KVIrc
+
+modules:
+  - name: kvirc
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
```

```suggestion
    buildsystem: cmake-ninja
```

cmake-ninja is prefered.

The prefix is automatically set.